.github/workflows/main.yaml

```diff
@@ -0,0 +1,24 @@
+name: main
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: windows-latest
+
+    steps:
+    # first enable longpaths
+    - name: Support longpaths
+      run: git config --system core.longpaths true
+    # Checkout the code.
+    - uses: actions/checkout@v3
+      # For the GitVersion to execute successfully, we need the checkout to fetch all gistory for all tags and branches (with fetch-depth: 0).
+      with:
+        fetch-depth: 0
+    # Use the compose file action.
+    - uses: ./.github/actions/compose_file
+    # TODO: add logic to version and add to a tag/release  
```

Comment:
I think this comment should be in de compose_file action. Since feature versions should also have this comment in them from which version and feature build the xem was created.

Reply:
moved